content/docs/rfcs/32/README.md

```diff
@@ -1,14 +1,15 @@
 ---
 slug: 32
-title: 32/RLN
+title: 32/RLN-V1
 name: Rate Limit Nullifier
-status: raw
+status: stable
 editor: Blagoj Dimovski <blagoj.dimovski@yandex.com>
```

Comment:
Was previously discussed that the editor should be someone actively updating the rfc(ref: https://rfc.vac.dev/spec/1/#editorial-control). Can we move Blagoj to the contributors section, and add @curryrasul as the main editor?